jenkins/pipelines/ci/tidb/tidb_ghpr_check_2.groovy

```diff
@@ -83,24 +84,64 @@ if (!isNeedGo1160 && ghprbTargetBranch.startsWith("release-")) {
 }
 if (isNeedGo1160) {
     println "This build use go1.16"
-    GO_BUILD_SLAVE = "build_go1160_memvolume"
-    GO_TEST_SLAVE = "test_go1160_memvolume"
+    pod_go_docker_image = "hub.pingcap.net/jenkins/centos7_golang-1.16:latest"
 } else {
     println "This build use go1.13"
 }
-println "BUILD_NODE_NAME=${GO_BUILD_SLAVE}"
-println "TEST_NODE_NAME=${GO_TEST_SLAVE}"
+println "use image=${pod_go_docker_image}"
 
-def buildSlave = "${GO_BUILD_SLAVE}"
-def testSlave = "${GO_TEST_SLAVE}"
+// def buildSlave = "${GO_BUILD_SLAVE}"
+// def testSlave = "${GO_TEST_SLAVE}"
 
 def sessionTestSuitesString = "testPessimisticSuite"
 
+def run_with_pod(Closure body) {
+    def label = "jenkins-check-2-" + UUID.randomUUID().toString()
```

Comment:
how about using `BUILD_NUMBER` to make pod label unique. Like this
`def label = "jenkins-check-2-${BUILD_NUMBER}"`